script/hassfest/requirements.py

```diff
@@ -42,6 +42,10 @@
     "urllib3": "SemVer",
     "yarl": "SemVer",
 }
+PACKAGE_CHECK_PREPARE_UPDATE: dict[str, int] = {
+    # Pandas will only fully support Python 3.14 in v3.
+    "pandas": 3,
```

Comment:
I was thinking if it's better to use an AwesomeVersion here, but I guess we don't need that at the moment.